packages/agents/src/agents/Scripter/Scripter.ts

```diff
@@ -84,9 +82,9 @@ If a goal has been achieved or failed, you will call the agent_onGoalAchieved or
         "Assistant, you seem to be looping. Try calling findScript, or exiting by calling agent_onGoalAchieved or agent_onGoalFailed.",
       functions: [
         new CreateScriptFunction(agentContext.scripts),
-        new ExecuteScriptFunction(agentContext.client, agentContext.scripts, agentContext.globals),
+        new ExecuteScriptFunction(agentContext.client, agentContext.scripts),
         new FindScriptFunction(agentContext.scripts),
-        new ReadVariableFunction(agentContext.globals),
+        new ReadVariableFunction(),
```

Comment:
nit: should we remove this function from here? iiuc it is added by default right

Reply:
Yep, but we have logic to handle this very situation where an agent manually adds the function. It won't get added twice.